lisa/energy_model.py

```diff
@@ -669,7 +674,7 @@ def estimate_from_cpu_util(self, cpu_utils, freqs=None, idle_states=None):
         return self._estimate_from_active_time(cpu_active_time,
                                                freqs, idle_states, combine=True)
 
-    def get_optimal_placements(self, capacities):
+    def get_optimal_placements(self, capacities, capacity_margin_pct=20):
```

Comment:
What about having the default value in `energy_model` be 0 (i.e. not used), and only keep the 20 for the `eas_behaviour` tests?

Reply:
Right, that sounds like a good idea.